build_tools/docker/cmake/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# An image for building IREE using CMake.
+
+# Build using:
+# docker build --tag gcr.io/iree-oss/cmake build_tools/docker/cmake/
+
+# Run using:
+# docker run -it --rm --entrypoint bash \
+#   --volume /home/phoenix/iree/fork:/usr/src/iree/ \
+#   gcr.io/iree-oss/cmake
+
+# Build and test IREE.
+# cmake -G Ninja -B build/ \
+#   -DCMAKE_C_COMPILER=clang \
+#   -DCMAKE_CXX_COMPILER=clang++ \
+#   -DIREE_BUILD_PYTHON_BINDINGS=ON .
+# cmake --build build/
+# ./build_tools/cmake/test.sh
+
+
+# Set up the image and working directory.
+FROM ubuntu:18.04
+WORKDIR /usr/src/iree/
+
+RUN apt-get update
+
+# Update cmake to v3.13+, which is ahead of apt-get's version (3.10.2).
+# Install dependencies, including an old version of cmake to bootstrap.
+RUN apt-get install -y clang cmake libssl-dev wget
+RUN mkdir ./cmake_install \
+  && cd cmake_install \
+  && wget https://github.com/Kitware/CMake/releases/download/v3.17.2/cmake-3.17.2.tar.gz \
```

Comment:
We support cmake >=3.13, so this build should probably use 3.13

Reply:
I'll change this to use the latest version of `3.13` - `3.13.5`